### PR TITLE
AT_01.01_008 | Verify Jenkins Icon Functionality

### PR DESCRIPTION
**Pull request by Paul-Korz**
https://trello.com/c/sm8HO0Ry/313-at101008-verify-jenkins-icon-functionality

### CODE REVIEW

**Review by OlyaPro — APPROVED** [review]

**Review by AnaG88 — COMMENTED** [review]
1. Redirection to the homepage should be from another page
2. Add check for the location of 'Jenkins' icon

File: cypress/e2e/header.cy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/// <reference types="cypress" />
+
+describe('<Header> Head Icon', ()=>{
```
Comment (Rom-T) [inline cypress/e2e/header.cy.js:3]: Please rename describe to 'Header | Head Icon'

Comment (Paul-Korz) [reply cypress/e2e/header.cy.js]: Done.

**Comment by Paul-Korz** [comment]
> 1. Redirection to the homepage should be from another page
> 2. Add check for the location of 'Jenkins' icon

First step - Done. 
Second step - I can't solve the problem yet. Later I will write a new test case for this.

**Review by OlyaPro — DISMISSED** [review]

**Review by Tatsiana-Nazarava — APPROVED** [review]

**Review by OlyaPro — APPROVED** [review]

**Pull request merged**